.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -101,15 +104,13 @@ jobs:
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt ${{ matrix.pytorch-version }}
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/examples.txt ${{ matrix.pytorch-version }}
 
-    - name: Install dependencies
-      env:
-        PACKAGE_NAME: pytorch
-        FREEZE_REQUIREMENTS: 1
+    - name: Install package & dependencies
       run: |
-        # install PL and optional dependencies for testing
+        # setting pre-release flag
         pre_option=$(python -c "print('--pre' if '${{matrix.release}}' == 'pre' else '')" 2>&1)
+        # install PL and optional dependencies for testing
         url=https://download.pytorch.org/whl/$(python -c "print('test/cpu/torch_test.html' if '${{matrix.release}}' == 'pre' else 'cpu/torch_stable.html')" 2>&1)
-        pip install -e . -r requirements/pytorch/test.txt -r ./requirements/pytorch/extra.txt $pre_option -f ${url}
+        pip install -e .[test,extra] -f ${url}  # $pre_option
```

Comment:
Same here. `$pre_option` is commented out. Is it on purpose so that we can work on any test failures in follow-up PRs?